pkg/agent/util/net_windows.go

```diff
@@ -106,6 +115,14 @@ func RemoveManagementInterface(networkName string) error {
 	return err
 }
 
+// ConfigureMACAddress set specified MAC address on interface
```

Comment:
Nit: add "." at the end of the sentence.

Reply:
Done